azure-pipelines.yml

```diff
@@ -109,6 +109,11 @@ stages:
             pytest: false
 
           - linux: py38-online
+            cache_dirs:
+            - key: 'sample_data'
+              path: .tox/py38-online/sample_data
```

Comment:
The sample data shouldn't be needed for anything other than docs (I hope).

Reply:
My `sunpy/.tox/py39-online/sample_data` has 65M of data so does that not mean it's spending time downloading the sample data? (And therefore can be cached.) If it's not needed, should the `online` tox environment be configured to not download sample data?

Reply:
Interesting.

Reply:
Ah I was incorrect, our narrative documentation uses sample data, which is tested in the online build, so yep, this would be very useful here and in the docs build (they should share the same cache id, and very rarely need changing).